src/main/kotlin/no/nav/familie/ef/mottak/service/Feltformaterer.kt

```diff
@@ -15,14 +15,14 @@ import java.util.Locale
 
 object Feltformaterer {
     /**
-     * Håndterer formatering utover vanlig toString for endenodene
+     * Håndterer formatering utover vanlig toString for endenodenee
      */
     fun mapEndenodeTilUtskriftMap(entitet: Søknadsfelt<*>): VerdilisteElement = mapTilVerdiListeElement(entitet)
 
     fun genereltFormatMapperMapEndenode(entitet: Søknadsfelt<*>): VerdilisteElement? {
         // skal ekskluderes
-        if (entitet.label == "Jeg har sendt inn denne dokumentasjonen til Nav tidligere" &&
-            entitet.verdi.toString() == "false"
+        if ((entitet.label == "Jeg har sendt inn denne dokumentasjonen til Nav tidligere" &&
+            entitet.verdi.toString() == "false") || (entitet.label == "Født" && entitet.verdi.toString() == "true")
```

Comment:
Ville trukket ut denne sjekken til en egen variabel, eks. `entitetSkalEkskluderes` evt. `skalEkskluderes`  - for bedre lesbarhet :)